Taskfile.yml

```diff
@@ -139,10 +139,12 @@ tasks:
       DEBUG: '{{.DEBUG | default "False"}}'
     cmds:
       - task: py:update
-      - pipenv run python -c \
+      - |
```

Comment:
I have a similar but different fix in my PR among other standardization

https://github.com/SeisoLLC/easy_infra/pull/279/files

I'd undo this and just rebase on `main` after my fix goes in.  It's needed to fix releasing anyway, so it needs to happen first